app/services/battle_service.rb

```diff
@@ -208,6 +216,47 @@ def abandon_battle(battle_id)
     broadcast_cable(battle_message)
   end
 
+  def update_battle_elo(battle)
+    axisElo = 0
+    alliedElo = 0
+    for player in battle.battle_players
+      if player.side == "axis"
+        axisElo += player.player_elo
+      else
+        alliedElo += player.player_elo
+      end
+    end
+    axisElo = axisElo / battle.size
+    alliedElo = alliedElo / battle.size
+    
+    if battle.size == 4
+      minDiff = 35
+    else
+      minDiff = 50
+    end
+
+    battle.diff = (axisElo - alliedElo).abs
+
+    if battle.diff < 10
+      battle.balance = "Perfect"
+    elsif battle.diff < minDiff
+      battle.balance = "Balanced"
+    elsif axisElo > alliedElo
+      battle.balance = "Axis favoured"
+    else
+      battle.balance = "Allied favoured"
+    end
```

Comment:
Let's only deal with elo diff integer values in the backend. The front end can translate that into a display string value.